scalableminds/webknossos
# fix empty airbrake config for dev

The latest update for airbrake made config values mandatory, which is why wk didn't start locally in dev mode. This PR replaces the empty strings with dummy values.

### URL of deployed dev instance (used for testing):
- https://fixemptyairbrakeconfigfordev.webknossos.xyz

### Steps to test:
- provoke error on dev instance (e.g., type into select box in user view) and see whether that issue is reported to airbrake

------
